tests/indices/vector_store/test_base.py

```diff
@@ -405,6 +413,7 @@ def test_query_and_similarity_scores(
     _mock_init: Any,
     _mock_predict: Any,
     _mock_total_tokens_used: Any,
+    _mock_split_text_overlap: Any,
```

Comment:
do we still need to keep _mock_split_text for all these tests if we now have a _mock_split_text_overlap?

Reply:
Yeah split_text is still used in a lot of places.